tests/expedition/test_ship_config.py

```diff
@@ -0,0 +1,168 @@
+from pathlib import Path
+
+import pytest
+
+from virtualship.expedition.schedule import Schedule
+from virtualship.expedition.ship_config import ConfigError, ShipConfig
+from virtualship.utils import get_example_config, get_example_schedule
+
+expedition_dir = Path("expedition_dir")
+
+
+@pytest.fixture
+def schedule(tmpdir):
+    out_path = tmpdir.join("schedule.yaml")
+
+    with open(out_path, "w") as file:
+        file.write(get_example_schedule())
+
+    schedule = Schedule.from_yaml(out_path)
+
+    return schedule
+
+
+@pytest.fixture
+def schedule_no_xbt(tmpdir):
+    out_path = tmpdir.join("schedule.yaml")
+
+    with open(out_path, "w") as file:
+        file.write(get_example_schedule())
+
+    schedule = Schedule.from_yaml(out_path)
+
+    for waypoint in schedule.waypoints:
+        if waypoint.instrument and any(
+            instrument.name == "XBT" for instrument in waypoint.instrument
+        ):
+            waypoint.instrument = [
+                instrument
+                for instrument in waypoint.instrument
+                if instrument.name != "XBT"
+            ]
+
+    return schedule
+
+
+@pytest.fixture
+def ship_config(tmpdir):
+    out_path = tmpdir.join("ship_config.yaml")
+
+    with open(out_path, "w") as file:
+        file.write(get_example_config())
+
+    ship_config = ShipConfig.from_yaml(out_path)
+
+    return ship_config
```

Comment:
```suggestion
@pytest.fixture
def ship_config(tmp_path):
    tmp_path.write(get_example_config())
    return ShipConfig.from_yaml(out_path)
```

Reply:
I don't think your suggestion works, as `tmp_path` is not a file name.